recipes/pysmi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pysmi" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "7b496a0831cc3909a6f2467b0852b3a87074d9f95949214e8120e014708c118b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
```

Comment:
It looks like the `setup.py` will use `setuptools` if available, so let's add that requirement:

https://github.com/etingof/pysmi/blob/master/setup.py#L63-L66